docs/theory.rst

```diff
@@ -30,7 +30,7 @@ Quantum Bit Simulation
 Simulating a quantum bit takes advantage of the probabilistic nature of the measurement result of a quantum bit and records strictly the complex number probability for each of the bits in a coherent state.  Thus, a two qubit system which can be expressed through the following equation:
```

Comment:
Firstly, your docs are generally good and accessible, but we need to clarify a few points of rigor.

The quantum state and its simulation actually store state "amplitudes," as in "amplitudes of the Schroedinger wave equation." An amplitude can be unambiguously specified in terms of probability along with _phase_. (The amplitude is actually the square root of the probability times a complex phase factor.)

The state representation isn't probabilistic. Up to a random overall phase factor, the unitary evolution of state is actually totally deterministic. Only measurement breaks unitarity, hence measurement can be probabilistic.